config/environments/production.rb

```diff
@@ -26,10 +26,10 @@
 
   # Compress JavaScripts and CSS.
   config.assets.js_compressor = :uglifier
-  config.assets.css_compressor = :sass
+  # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = true
+  config.assets.compile = false
```

Comment:
note that this reverses https://github.com/ehmorris/buy.edwin.pizza/commit/5019d43feec36f586877aa06e46f5eb71aa5a562 — it was an explicit change but I don't remember why I did it